docs/content/AutoGPT/setup/index.md

```diff
@@ -79,27 +79,22 @@ See the [user guide](../usage.md) for further instructions.
 [show hidden files/Windows]: https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5
 [show hidden files/macOS]: https://www.pcmag.com/how-to/how-to-access-your-macs-hidden-files
 
+
 ## Setting up LLM providers
```

Comment:
You can choose to use any of the following LLM providers. Each of them needs their own details to setup

Reply:
done